armi/reactor/components/component.py

```diff
@@ -1240,6 +1240,12 @@ def getMicroSuffix(self):
         return self.parent.getMicroSuffix()
 
 
+class ShapedComponent(Component):
```

Comment:
This had to end up not in __init__ to avoid circular imports inside basic shapes. Open to other locations.